weblogic-azure-vm/arm-oraclelinux-wls-admin/src/main/arm/mainTemplate.json

```diff
@@ -432,6 +432,54 @@
             "description": "Select appropriate VM Size as per requirement"
          }
       },
+      "virtualNetworkNewOrExisting": {
+         "type": "string",
+         "defaultValue": "new",
+         "allowedValues": [
+            "new",
+            "existing"
+         ],
+         "metadata": {
+            "description": "Specify whether to create a new or existing virtual network for the VM."
+         }
+      },
+      "virtualNetworkName": {
+         "type": "string",
+         "defaultValue": "jbosseap-vnet",
```

Comment:
Should not be jbosseap.  Should be **wls-vnet**

Reply:
Addressed